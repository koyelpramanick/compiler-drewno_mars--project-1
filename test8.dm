"testing all \t of these \n inside a \" string   "
890823488934201
"can you escape a \k?"
"illegal char inside string &*^^&*(("
1+2-(87*5)/2
"here's an illegal char: " &
"let's not finish this string