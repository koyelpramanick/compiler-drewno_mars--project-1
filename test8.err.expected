FATAL [2,1]-[2,16]: Integer literal overflow
FATAL [3,1]-[3,23]: String literal with bad escape sequence detected
FATAL [6,28]-[6,29]: Illegal character &
FATAL [7,1]-[7,30]: Unterminated string literal detected
